src/MobileBookingHeader/MobileBookingDetail.js

```diff
@@ -54,7 +49,13 @@ const MobileBookingControls = (props: MobileBookingControlsProps) => (
       href={props.manageBookingURL}
       className="manageBookingButton"
     >
-      Manage My Booking
+      <Button
+        block
+        onClick={() => {}}
```

Comment:
it should probably open new tab `onClick`

Reply:
I wrapped it in a link instead

Reply:
btw why `noopener noreferrer`?

Reply:
That's a good question! :joy: Actually the code for the link was here so I just reused that. I know this is a very bad explanation given that I created the entire component.

Reply:
Ah! I took it from here: https://github.com/kiwicom/smart-faq/blob/d438e98488cd0d0ff3c659c59027690ddc104076/src/SingleBookingPage/BookingDetail.js#L112

Reply:
it should be there, we want to track it, right? 🙂

Reply:
in that case let's do it in `onClick`

Reply:
Like this? BTW why is the surge link broken? :thinking:

Reply:
your branch names are too big 😂, surge couldn't handle it

Reply:
:joy: